### ISSUE TITLE
Grbl or marlin 

### ISSUE THREAD

**Issue by nesquik011** [ThingEngineer/Laser-XY-Scanner#1]
hello can i use it with Grbl or marlin ?  to make 3D printer with laser

**Comment by ThingEngineer** [comment]
Hey, I'm guessing this is MostafaM44. I posted a response on the Instructable:
https://www.instructables.com/id/Low-Cost-DIY-Stepper-Motor-Laser-XY-Scanner-Cutter/

**Comment by nesquik011** [comment]
yes sir thats me 👍  ha ha  nice guess , i want to make it work , i have been searching from 2016 to make this work but no luck yet and i decided to contact you , you the prof here and know more than me and could guide me in the right way , i am so sorry if i was bothering you :/
